nautobot/core/settings.yaml

```diff
@@ -0,0 +1,1636 @@
+    "is_constance_config": true
+    "type": "string"
+  "BRANDING_FILEPATHS":
+    "default":
+      "favicon": ~
+      "header_bullet": ~
+      "icon_16": ~
+      "icon_32": ~
+      "icon_180": ~
+      "icon_192": ~
+      "icon_mask": ~
+      "logo": ~
+      "nav_bullet": ~
+    "description": >-
+      A set of filepaths relative to the [`MEDIA_ROOT`](#media_root) which locate image assets used for
+      custom branding. Each of these assets takes the place of the corresponding stock Nautobot asset.
+      This allows for, for instance, providing your own navbar logo and favicon.
+      If a custom image asset is not provided for any of the above options, the stock Nautobot asset is used.
+    "properties":
+      "favicon":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "Browser favicon"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_FAVICON"
+      "header_bullet":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "Bullet image used for various view headers"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_HEADER_BULLET"
+      "icon_16":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "16x16px icon"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_ICON_16"
+      "icon_180":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "180x180px icon - used for the apple-touch-icon header"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_ICON_180"
+      "icon_192":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "192x192px icon"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_ICON_192"
+      "icon_32":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "32x32px icon"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_ICON_32"
+      "icon_mask":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "Mono-chrome icon used for the mask-icon header"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_ICON_MASK"
+      "logo":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "Navbar logo"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_LOGO"
+      "nav_bullet":
+        "$ref": "#/definitions/relative_path"
+        "default": ""
+        "description": "Bullet image used for nav menu headers"
+        "environment_variable": "NAUTOBOT_BRANDING_FILEPATHS_NAV_BULLET"
+    "type": "object"
+  "BRANDING_PREPENDED_FILENAME":
+    "default": "nautobot_"
+    "description": "Defines the prefix of the filename when exporting to CSV/YAML or export templates."
+    "environment_variable": "NAUTOBOT_BRANDING_PREPENDED_FILENAME"
+    "type": "string"
+    "version_added": "1.3.4"
```

Comment:
not seeing the version added annotation for BRANDING_FILEPATHS in the new docs

Reply:
Sharp eyes! Will fix.